Assets/Scripts/Model/Cards/Corp/AdvancedAssemblyLines.cs

```diff
@@ -19,7 +19,7 @@ public class AdvancedAssemblyLines : Card
         override public int InfluenceCost => 2;
         override public ICost PlayCost => new CorpCreditCost(1);
         override public IEffect Activation => new AdvancedAssemblyLinesActivation(this);
-        override public IType Type => new Asset();
+        override public IType Type => new Asset(); 
```

Comment:
Now we can remove all diff from specific cards thanks to the defaults.
It can be done e.g. via `git checkout origin/master -- Assets/Scripts/Model/Cards/Corp/`.

PS. Generally, I'm quite picky about Git history cleanliness:
* messages: https://chris.beams.io/posts/git-commit/
* linear history
* atomic commits

But I know it's a bunch of extra effort (especially learning the `git rebase --interactive`.
So I could cherry-pick your commits, restructure them a little and merge then. WDYT?

Reply:
Absolutely: it is your project, so please do what you think it is best 😄

Reply:
Ok, continued in https://github.com/dagguh/data-hunt/pull/2027